src/components/clock/clock.scss

```diff
@@ -1,58 +1,82 @@
-.capitalize {
-	&:first-letter {
-		text-transform: capitalize;
-	}
-}
-
-.clock {
-	position: relative;
-	border-radius: 50%;
-	border: 3px solid black;
-	box-sizing: border-box;
-	width: 100px;
-	height: 100px;
-
-	* {
-		background-color: black;
-	}
+.clock-container {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
 
-	.mark-12 {
-		width: 2px;
-		height: 6px;
-		left: calc(50% - 1px);
-		position: absolute;
-	}
-	
-	.mark-3 {
-		width: 6px;
-		height: 2px;
-		right: 0%;
-		top: calc(50% - 2px);
-		position: absolute;
-	}
+	.capitalize {
+		max-width: 80px;
+		display: block;
+		text-align: center;
+		margin-top: 20px;
 
-	.mark-6 {
-		width: 2px;
-		height: 6px;
-		left: calc(50% - 1px);
-		bottom: 0%;
-		position: absolute;
+		&::first-letter {
+			text-transform: capitalize;
+		}
 	}
 
-	.mark-9 {
-		width: 6px;
-		height: 2px;
-		left: 0%;
-		top: calc(50% - 2px);
-		position: absolute;
-	}
+	$size: 4px;
```

Comment:
@sinejespersen I'm just wondering - would this variable naming maybe be too general? When this styling becomes a part of the dpl-design-system, and it's imported by base.scss from its own file, it could pose some problems.

Reply:
Yes valid input thanks, I've changed it and changed the scss to be bem in structure (dusting off my bem-skills as well...) and prefixing the class with "dpl" as per your guidelines...

Reply:
@sinejespersen oh yeah, about the dpl prefix - I'm not quite sure how that works. These dpl- prefix rules come from before my age, and I haven't dwelled in the topic yet.
I made all my design in the design system, and then just imported it. We really have to help to make sure you can do that as well 🙏🏼.